x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/formatted_field.tsx

```diff
@@ -202,11 +202,11 @@ const FormattedFieldValueComponent: React.FC<{
             </EuiFlexGroup>
           }
         >
-          <>{value}</>
+          <span data-test-subj={`formatted-field-${fieldName}`}>{value}</span>
```

Comment:
Did you verify that this has no impact to layout?

Reply:
looked good in the cypress view